.gitignore

```diff
@@ -34,3 +34,5 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+.idea/
+.rubocop.yml
```

Comment:
No need to ignore rubocop.yml, you can commit it in order for all (potential) developers in the team to keep the same style. It may also be needed for the CI